### ISSUE TITLE
Usage of  /addr/streetAdressLine in CDA

### ISSUE THREAD

**Issue by Remcovanderwerf** [scandihealth/lpr3-docs#135]
**link:**  
http://lpr-art-decor.westeurope.cloudapp.azure.com:8080/art-decor/decor-templates--lpr-?id=1.2.208.176.7.1.10.132&effectiveDate=dynamic

**CDA path:**  
/ClinicalDocument/recordTarget/patientRole/addr/streetAddressLine

**Background:**
in the CDA template the use of streetAddressLine is the following:
hl7:streetAddressLine | ADXP | 1 … 4 | R | SHALL contain at least one and not more than 4 streetAddressLine.

In other words; the element <streetAddressLine> can be repeated between 1 and 4 times. The LPR3 messages contains a possibility of 3 <adresseLinje#>. The mapping is possible, but:

**Question:**  
The original CDA R2 standard states the <streetAddressLine> can only occur min: 1 and max: 1.
Does the CDA R2 standard scheme needs to be edited to min: 1 and max: 4?

**Comment by jkiddo** [comment]
The CDA R2 schemas states that the min use and max use of addr is unbounded. Templates used in LPR3 restricts this. Where do you see that the original CDA R2 spec restricts this to 1..1?

![image](https://user-images.githubusercontent.com/211696/41526618-0a3afd6c-72e5-11e8-8790-5ea5ac7d92c8.png)

**Comment by Remcovanderwerf** [comment]
I am talk talking about the element <streetAddressLine> under patientRole/addr/

![image](https://user-images.githubusercontent.com/39521104/41532368-02038846-72f7-11e8-81c0-bd26ec9da3d1.png)

Do you want me to map each <adresseLinje> from the LPR3 message to a individual <addr> element?
Is every <adresseLinje> in the <Bopael> element an individual adress of the patient in context?
In this case i would expect multiple uses of the <Bopael> element in the LPR3 message.

 or are do these <adresseLinje> all belong to 1 adress?

**Comment by jkiddo** [comment]
I am not sure if I'm understanding you correctly. See http://svn.medcom.dk/svn/drafts/Standarder/HL7/CDA%20Header/Dokumentation/DK-CDA-Header-v1.1.pdf page 15 for an example.

**Comment by Remcovanderwerf** [comment]
From my point of view, this example of usage of the CDA R2 template is false. 
Following the CDA R2 standard, the usage of the streetAdressLine element is min 1, max 1 (check my screenshot), not max 4.
Under the element < addr > there can only be 1 occurance of < streetAddressLine >
There can be multiple < addr > elements used.
Please advice.

**Comment by jkiddo** [comment]
Your screenshot does not reflect the standard - it reflects a view in your system. Please pinpoint in the standard where it says that the stated use is illegal / not allowed / that your interpretation is correct.

**Comment by jkiddo** [comment]
While this is not from the standard directly - it is from the CDA Book:

![image](https://user-images.githubusercontent.com/211696/41546153-30c7bbe0-731d-11e8-8f00-3ca1baf46172.png)

**Comment by jkiddo** [comment]
![image](https://user-images.githubusercontent.com/211696/41547705-3cf96388-7321-11e8-9100-69626e018018.png)

**Comment by jonigkeit** [comment]
Directly from the CDA R2 Specification - HL7 Version 3 Data Types - Abstract Specification - section 2.21.5

> Example 14
```xml
<addr use="WP">
  <streetAddressLine>1050 W Wishard Blvd</streetAddressLine>,
  <streetAddressLine>RG 5th floor</streetAddressLine>,
  <city>Indianapolis</city>,
  <state>IN</state>
  <postalCode>46240</postalCode>
</addr>
```

As @jkiddo says, please familiarize yourself with the appropriate standards before asking question.

**Comment by jonigkeit** [comment]
@Remcovanderwerf if you have no further comments please close this issue.
